src/store/__mocks__/RelayStore.js

```diff
@@ -9,8 +9,8 @@
 
```

Comment:
Hmm, lots of tests are relying on the mocks here though. Do you think those should be moved into `RelayContext` then?